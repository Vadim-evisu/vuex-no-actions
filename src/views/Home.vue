<template>
  <div>
    <SearchBox />
    <p>Go ahead and make a search</p>
  </div>
</template>

<script lang="ts">
import Vue from 'vue';

import SearchBox from '@/components/SearchBox.vue';

export default Vue.extend({
  name: 'Home',
  components: {
    SearchBox,
  },
});
</script>
