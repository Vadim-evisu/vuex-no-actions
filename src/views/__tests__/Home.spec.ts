import { shallowMount } from '@vue/test-utils';
import Home from '@/views/Home.vue';

describe('Home component', () => {
  const wrapper = shallowMount(Home);
  it('matches snapshot', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
