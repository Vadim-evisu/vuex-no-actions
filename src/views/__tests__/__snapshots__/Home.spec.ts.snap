// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component matches snapshot 1`] = `
<div>
  <searchbox-stub />
   
  <p>
    Go ahead and make a search
  </p>
</div>
`;
